### ISSUE TITLE
Doesn't load on Ubuntu

### ISSUE THREAD

**Issue by daemonspudguy** [mctofu/deadbeef-library#4]
I built this plugin because it intrigued me, and when I did, I got this error when starting DeaDBeeF:

> loading plugin /home/daemonspudguy/.local/lib/deadbeef/ddb_misc_libbrowser_GTK3.so
dlopen error: /home/daemonspudguy/.local/lib/deadbeef/ddb_misc_libbrowser_GTK3.so: undefined symbol: gtk_tree_path_free
trying /home/daemonspudguy/.local/lib/deadbeef/ddb_misc_libbrowser_GTK3.fallback.so...
plugin ddb_misc_libbrowser_GTK3.so not found or failed to load

**Comment by mctofu** [comment]
Could you share which versions of the gtk3 libs you have installed?

```
$ apt list libgtk-3-*
Listing... Done
libgtk-3-0/stable,now 3.24.5-1 amd64 [installed,automatic]
libgtk-3-bin/stable,now 3.24.5-1 amd64 [installed,automatic]
libgtk-3-common/stable,now 3.24.5-1 all [installed,automatic]
libgtk-3-dev/stable,now 3.24.5-1 amd64 [installed]
libgtk-3-doc/stable 3.24.5-1 all
```

It would also be useful to know what version of Ubuntu & DeaDBeeF you are running.

Have you set DeaDBeeF to use the GTK3 GUI plugin? This plugin doesn't support GTK2.

**Comment by daemonspudguy** [comment]
Ubuntu 21.04 (technically it's Pop!_OS)
DeaDBeeF 1.8.7
Whatever the latest GTK3 libs on Ubuntu 21.04 are.

**Comment by mctofu** [comment]
I was able to reproduce this on an Ubuntu 20.04 VM. I tweaked the build script to fix the issue in https://github.com/mctofu/deadbeef-library/pull/5. That should fix things for 21.04 as well but feel free to re-open if a rebuild with the latest doesn't fix this issue for you.

**Comment by daemonspudguy** [comment]
Of course this gets patched while I'm visiting a relative in Virginia. Lucky me. I won't be able to test it on 21.04 until the beginning of August.
